common/Android.mk

```diff
@@ -62,6 +62,11 @@ LOCAL_CPPFLAGS += \
 	-DDISABLE_CURSOR_PLANE
 endif
 
+ifeq ($(strip $(HWC_DISABLE_VA_DRIVER)), true)
+LOCAL_CPPFLAGS += \
```

Comment:
Please don't compile any libva related source files in this case.

Reply:
we can do this. but just removing libva source files are not enough. there is still other VA coupled code pieces over the other source files. we need to add more micro #ifdef DISABLE_VA over the pieces which is ugly.